.github/workflows/go.yml

```diff
@@ -0,0 +1,138 @@
+name: Fabric Interop CC and Sample Chaincode Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test_interopcc_github:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: Build
+      run: go build -v ./...
+      working-directory: core/network/fabric-interop-cc/contracts/interop
+      
+    - name: Test
+      run: go test -v ./...
+      working-directory: core/network/fabric-interop-cc/contracts/interop
+      
+  test_interopcc_local:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+        
+    - name: Local Setup
```

Comment:
We are using the imported protos here, right? Not locally built ones?

Reply:
The place where you commented, here we are using locally built ones.
There's another job above this, which uses imported ones.

Reply:
Actually I'm assuming the person updating protos, should always rebuild protos-go before checking in, and so that, that's always upto date whatever is in `common/protos-go`.